app/Resources/views/events/index.html.twig

```diff
@@ -19,6 +21,7 @@
                 </thead>
                 <tbody>
                 {% for event in events %}
+                {% if event.rejected == 0 %}
```

Comment:
identation du if dans le for ! (ne devrait pas arriver si tu utilisais PHPStorm pour vérifier la syntaxe avant chaque commit